### ISSUE TITLE
Relay shortcuts don't use Connection _nullable configuration

### ISSUE THREAD

**Issue by such** [rmosolgo/graphql-ruby#3423]
**Describe the bug**

When using the relay shortcuts to get the connection type of a type (eg `MyType.connection_type`), the `{edges|edge|node}_nullable` configuration of the Connection declared as `connection_type_class` is not used.

**Versions**

`graphql` version: 1.12.6
`rails`: 6.1.3.1

**GraphQL schema**

```ruby
module Types
  module Base
    # Base::ConnectionObject is the Base connection object
    class ConnectionObject < GraphQL::Types::Relay::BaseConnection
      edges_nullable(false)
      edge_nullable(false)
      node_nullable(false)      
    end
  end
end

module Types
  module Base
    # Base::Object overrides the default connection type class
    class Object < GraphQL::Schema::Object
      connection_type_class(ConnectionObject)
    end
  end
end

module Types
  class Bid < Types::Base::Object
  end
end

module Types
  class Auction < Types::Base::Object
    field :bids, Bid.connection_type
  end
end
```

**Expected behavior**

We would expect the `nodes` field of `bids` not to include nullable objects.

**Additional context**

It would seem the issue lies in this piece of code in `lib/graphql/schema/member/relay_shortcuts.rb`:

```
      def connection_type
          @connection_type ||= begin
            conn_name = self.graphql_name + "Connection"
            edge_type_class = self.edge_type
            Class.new(connection_type_class) do
              graphql_name(conn_name)
              edge_type(edge_type_class)
            end
          end
        end
 ```

Because `{edges|edge|node}_nullable` is stored as an instance variable of `connection_type_class`, it is not inherited when the class is extended.

**Comment by rmosolgo** [comment]
These methods include a check for a superclass value, for example: 

https://github.com/rmosolgo/graphql-ruby/blob/1d24961ea94b747e6a92fd51a53cb105118145a7/lib/graphql/types/relay/connection_behaviors.rb#L103

However,  there was a bug in the implementation in `1.12.6`, fixed by #3386. I just released 1.12.7 which includes that fix, could you update to that version and let me know if it fixes the problem for you?

**Comment by rmosolgo** [comment]
Please let me know if you're still encountering this error after upgrading to 1.12.7!
